planner/core/planbuilder.go

```diff
@@ -2138,8 +2140,8 @@ func (b *PlanBuilder) buildShow(ctx context.Context, show *ast.ShowStmt) (Plan,
 		err := ErrSpecificAccessDenied.GenWithStackByArgs("SHOW VIEW")
 		b.visitInfo = appendVisitInfo(b.visitInfo, mysql.ShowViewPriv, show.Table.Schema.L, show.Table.Name.L, "", err)
 	case ast.ShowBackups, ast.ShowRestores:
-		err := ErrSpecificAccessDenied.GenWithStackByArgs("SUPER")
-		b.visitInfo = appendVisitInfo(b.visitInfo, mysql.SuperPriv, "", "", "", err)
```

Comment:
This is handled in the code here: https://github.com/pingcap/tidb/pull/22778/files#diff-99a8f0eee1a05220cf398e9af21bc220b8dff9281812d7df9504f955a6ab6b26R979-R987

TiDB will always* accept `SUPER` as a substitute for a Dynamic privilege (versus MySQL which will usually accept it). This addresses the bootstrap issue and compatibility issue. Long term I think this should diverge, but it would require changes to `visitInfo` to attach an `OR` condition (currently it can only accept `AND`).

\* There will be an exception with RESTRICTED_* dynamic privileges introduced in SEM.